data/lib/compat/compat.lua

```diff
@@ -1004,3 +1004,34 @@ end
 function Guild.removeMember(self, player)
 	return player:getGuild() == self and player:setGuild(nil)
 end
+
+function getPlayerInstantSpellInfo(cid, spellId)
+	local player = Player(cid)
+	if player == nil then
+		return false
+	end
+
+	local spell = Spell(spellId)
+	if spell == nil or not player:canCast(spell) then
+		return false
+	end
+
+	local spellInfo = {
```

Comment:
I meant return the table directly e.g. `return { stuff = 'in here' }` instead of assigning to `spellInfo` and `return spellInfo`